models/typescript/package-lock.json

```diff
@@ -0,0 +1,4691 @@
+{
```

Comment:
[suggestion]: We are using yarn.lock on other projects; we can use it also here for consistency.